api/modules/twitter/views.py

```diff
@@ -0,0 +1,91 @@
+import requests
+import requests_cache
+from datetime import timedelta
+
+from requests_oauthlib import OAuth1
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from api.models import City
+from api.modules.twitter.constants import TWITTER_CONSUMER_KEY, TWITTER_OAUTH_TOKEN_SECRET, TWITTER_OAUTH_TOKEN, \
+    TWITTER_CONSUMER_SECRET, TWITTER_TRENDS_URL, TWITTER_SEARCH_URL
+from api.modules.twitter.twitter_response import SearchTweetResponse
+
+hour_difference = timedelta(hours=1)
+requests_cache.install_cache(expire_after=hour_difference)
```

Comment:
Please test a case where you have 2 different cities
Ideally their trends should be different because that's what Twitter is giving, but because of this caching, we are rendering the cached results for each city

Reply:
Yes, It is working as expected.
For different cities, it is giving different result. For individual city it will give cached result which will be updated after hour.